### PR TITLE
Updated the title of playlist with the creator' name and institute as mentioned officially

### PR DESCRIPTION
**Pull request by ankita1964**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Updated Graph Theory Youtube courses in free-courses-en.md
### Why is this valuable (or not)?
Yes, it is valuable as it is widely used by students 
### How do we know it's really free?
Click on [this link](https://www.youtube.com/playlist?list=PLEAYkSg4uSQ2fXcfrTGZdPuTmv98bnFY5) and you ll be able to access and view the complete playlist
### For book lists, is it a book? For course lists, is it a course? etc.
It is a course
## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by ankita1964** [comment]
@eshellman Please review and merge! Thankyou

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -771,6 +771,7 @@
 
 ### Graph Theory
 
+* [Graph Theory](https://www.youtube.com/playlist?list=PLEAYkSg4uSQ2fXcfrTGZdPuTmv98bnFY5) - Graph Theory Lectures by Prof. Soumen Maity teaching at IISER
```
Comment (eshellman) [inline courses/free-courses-en.md:774]:
```suggestion
* [Graph Theory - Soumen Maity \| IISER - NPTEL](https://www.youtube.com/playlist?list=PLEAYkSg4uSQ2fXcfrTGZdPuTmv98bnFY5) - Graph Theory
```

Comment (eshellman) [reply courses/free-courses-en.md:774]: Don't invent titles! Use the title given by the resource.

Comment (ankita1964) [reply courses/free-courses-en.md]: Done

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -771,6 +771,7 @@
 
 ### Graph Theory
 
+* [Graph Theory](https://www.youtube.com/playlist?list=PLEAYkSg4uSQ2fXcfrTGZdPuTmv98bnFY5) - Graph Theory - Soumen Maity | IISER - NPTEL
```
Comment (eshellman) [inline courses/free-courses-en.md:774]:
```suggestion
* [Graph Theory](https://www.youtube.com/playlist?list=PLZ6w_oR8t_Pw4P3t5be9WMhj1ISH7yFWI) - Soumen Maity (Graph Theory)
```
We don't like 3rd party lists

**Pull request merged**